tests/all/integration/api/helpers/test_order.py

```diff
@@ -54,6 +56,120 @@ def test_should_delete_related_attendees(self):
             order = db.session.query(Order).filter(Order.id == obj.id).first()
             self.assertEqual(len(order.ticket_holders), 0)
 
+    def test_count_sold_tickets(self):
+        """Method to test the count query of sold tickets"""
+
+        with app.test_request_context():
+            # import logging
+            #
+            # logger = logging.getLogger('sqlalchemy.engine')
+            # logger.setLevel(logging.INFO)
+            ticket = TicketFactory()
+
+            first_completed_order = OrderFactory()
+            first_completed_order.status = 'completed'
+            second_completed_order = OrderFactory()
+            second_completed_order.status = 'completed'
+            third_completed_order = OrderFactory()
+            third_completed_order.status = 'completed'
+            fourth_completed_order = OrderFactory()
+            fourth_completed_order.status = 'completed'
+            fifth_completed_order = OrderFactory()
+            fifth_completed_order.status = 'completed'
+            sixth_completed_order = OrderFactory()
+            sixth_completed_order.status = 'completed'
```

Comment:
No need to create 5 completed orders. 1 is enough

Reply:
As you said here
`Create 2 attendees with no order ID, 6 with completed order, 1 with placed, 4 with initializing/initialized and not expired (created_at within order expiry time), 3 with initializing but expired created_at, 5 order status == expired`

You mean to create 1 completed order and link with 5 attendees or 1 completed order and 1 attendee with that?

Reply:
Yes

Reply:
This is yes of which part? @iamareebjamal

Reply:
1 order 6 attendees